### PR TITLE
#64 add filter kabkota_id

### LINKED ISSUE #64: [BUGFIX] Issue on get list
## Issue
Client cannot use filter and sorting in get list

## What should the behavior?
- Client should get list with filter by `text`, `name`, `tags` and `status`
- Client should get list and ordering by what client need

## Tasks
- [x] update query get list
- [x] update body request for get list

### PR DESCRIPTION
**Pull request by setiadijoe**
close #64 
add filter `kabkota_id` for get list post by actor's location who created the post

### CODE REVIEW

**Review by firmanJS — APPROVED** [review]

File: endpoint/request.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ type GetListUserPostRequest struct {
 	SortBy       string  `httpquery:"sort_by"`
 	OrderBy      string  `httpquery:"order_by"`
 	Search       *string `httpquery:"search"`
+	RegencyID    *int64  `httpquery:"kabkota_id"`
```
Comment (sandisunandar99) [inline endpoint/request.go:19]: just advice for naming this variable, coz this field kabkota_id, i think use "districtcity" more suitable than "RegencyID"

cmiwiw

Comment (firmanJS) [reply endpoint/request.go:19]: oh yeah +1

Comment (setiadijoe) [reply endpoint/request.go:19]: Regency is stand for `kabupaten / kota`
else District is stand for `kecamatan`
@sandisunandar99

**Review by firmanJS — APPROVED** [review]

File: endpoint/request.go (hunk 1 above)
Comment (sandisunandar99) [reply endpoint/request.go:19]: Regency = "daerah",, check2 on gslate :smile

**Review by sandisunandar99 — APPROVED** [review]
LGTM

**Pull request merged**